<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetting</class>
 <widget class="QDialog" name="DialogSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>209</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>191</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="radio_auto">
      <property name="text">
       <string>Auto</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radio_manual">
      <property name="text">
       <string>Manual</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>207</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="exitBtn">
      <property name="styleSheet">
       <string notr="true">color=rgb(204, 0, 0)</string>
      </property>
      <property name="text">
       <string>Đóng</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnSaveSetting">
      <property name="styleSheet">
       <string notr="true">color=rgb(78, 154, 6)</string>
      </property>
      <property name="text">
       <string>Lưu</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
